pkg/config/staging.go

```diff
@@ -0,0 +1,120 @@
+// Copyright 2024 "Google LLC"
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"crypto/md5"
+	"fmt"
+	"path/filepath"
+
+	"github.com/hashicorp/hcl/v2/hclsyntax"
+	"github.com/pkg/errors"
+	"github.com/zclconf/go-cty/cty"
+	"github.com/zclconf/go-cty/cty/function"
+)
+
+// Relative path from deployment group to the staging directory
+const StagingDir = "../.ghpc/staged"
+
+// Returns a map of absolute src to relative (to deployment folder) destination path
+func (bp Blueprint) StagedFiles() map[string]string {
+	if len(bp.stagedFiles) == 0 {
+		return nil
+	}
+
+	if bp.path == "" {
+		panic("blueprint doesn't have known path, can't resolve staged files to absolute paths")
+	}
+	res := map[string]string{}
+	for src, dst := range bp.stagedFiles {
```

Comment:
Should we log it somewhere how many stagedfiles found ? Or it is already being taken care of ?

Reply:
Nah, we will only log if some problem occurs (e.g. missing file)